src/components/Sidebar.tsx

```diff
@@ -0,0 +1,40 @@
+import { Bars3Icon, XMarkIcon } from "@heroicons/react/24/solid";
+import { useState } from "react";
+import Settings from "./Settings";
+
+// Represents a sidebar with settings options
+// This slides in and out depending on if the user clicks the hamburger icon. 
+
+const Sidebar = () => {
+  const [showSidebar, setShowSidebar] = useState(false);
+
+  return (
+    <>
+      {showSidebar ? (
+        <></>
```

Comment:
Unnecessary blank container? You can do `{!showSidebar && <Bars3Icon className="w-12 fill-warning-dark flex-none fill-white" onClick={() => setShowSidebar(!showSidebar)} />}`